frontend/package.json

```diff
@@ -5,7 +5,7 @@
   "type": "module",
   "scripts": {
     "dev": "vite",
-    "build": "tsc && vite build",
+    "build": "tsc && node --max_old_space_size=8192 ./node_modules/vite/bin/vite.js build",
```

Comment:
Adding this line here was found to have broken the docker build previously when I originally introduced it, mainly because at the time the docker build runs the node_modules folder doesn't yet exist..  I have added this change elsewhere in other places outside of the package.json so we wouldn't need it here, so I think you can revert this one and all should be good from a CI standpoint.

Reply:
We may also want to consider a doc update for the use of the `NODE_OPTIONS` variable when doing local development, since if we remove it here then it has to be added on the command line during the build.